build/pxc-entrypoint.sh

```diff
@@ -322,8 +326,8 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			CREATE USER 'operator'@'${MYSQL_ROOT_HOST}' IDENTIFIED BY '${OPERATOR_ADMIN_PASSWORD}' ;
 			GRANT ALL ON *.* TO 'operator'@'${MYSQL_ROOT_HOST}' WITH GRANT OPTION ;
 
-			CREATE USER 'xtrabackup'@'localhost' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
-			GRANT RELOAD,PROCESS,LOCK TABLES,REPLICATION CLIENT ON *.* TO 'xtrabackup'@'localhost';
+			CREATE USER 'xtrabackup'@'%' IDENTIFIED BY '${XTRABACKUP_PASSWORD}';
+			GRANT RELOAD,PROCESS,LOCK TABLES,REPLICATION CLIENT, DROP, DELETE, CREATE, INSERT, UPDATE PRIVILEGES ON *.* TO 'xtrabackup'@'%';
```

Comment:
have ever tested it? I see syntax error
```suggestion
			GRANT RELOAD,PROCESS,LOCK TABLES,REPLICATION CLIENT, DROP, DELETE, CREATE, INSERT, UPDATE ON *.* TO 'xtrabackup'@'%';
```